Note that table.find exists (#604)
Roblox/creator-docs: content/en-us/tutorials/fundamentals/coding-5/making-changes-to-arrays.md

## Searching for Values in an Array

> To find specific values in arrays, like the name of a winning player, you can code your own **search function** using for loops and if statements.
> 
> <Alert severity="info">
> There is already a builtin function for this: `table.find`
> </Alert>

Reviewer: ```suggestion
To find specific values in arrays, like the name of a winning player, use the `Library.table.find()` function. Alternatively, you can code your own search function using `for` loops and `if` statements.
```

Reviewer: Seems like table.find() is just much more straightforward than the alternative, so putting that first.

Also for your reference (if you haven't found it already), see https://github.com/Roblox/creator-docs/blob/main/STYLE.md#link-syntax for how API page links can be crafted within backticks as I've done for table.find() here.
